lib/rules/redux-actions-return-objects.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prohibits returning anything but objects from Redux actions',
+    },
+    fixable: false,
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          whiteList: {
+            default: [],
+            type: 'array',
+          },
+        },
+      },
+    ],
+  },
+
+  create(context) {
+    return {
+      'ReturnStatement'(node) {
```

Comment:
Is this.. declaring a function with a string..?

Reply:
Yup.

(The reason the convention is to declare them this way is that you can do more complicated paths like `'ReturnStatement[argument.type="Something"]'() {}`.)